deploy/README.md

```diff
@@ -0,0 +1,61 @@
+# Deploy on Google Cloud Platform
+
+### Setup
+
+
+* [Follow these instructions](https://github.com/DataBiosphere/data-explorer-indexers/tree/master/bigquery/deploy)
+to bring up Elasticsearch in GKE and index dataset. If you are indexing
+sensitive data, set up Elasticsearch only, deploy the app and ensure IAP is
+properly enabled here before indexing your Elasticsearch instance.
+
+* If you haven't done so already, set the default project:
+
+  `gcloud config set project PROJECT_ID`
+
+* Create the App Engine application:
+
+  `gcloud app create`
+
+* Set up routing rules
+
+  `gcloud app deploy dispatch.yaml`
+
+### Deploy the UI Server
+
+* Deploy:
+
+  `cd ../ui && gcloud app deploy && cd ../deploy`
+
+* Navigating to the UI URL should display an empty page for now, as the API
+server has not yet been set up.
+
+### Deploy the API Server
+
+* Ensure that `api/config` contains the correct dataset and facet fields for
```

Comment:
contains the config files for your dataset.

In the future, there will be more files, so let's not list the files here.